tests/utils/test_helpers.py

```diff
@@ -169,6 +171,22 @@ def test_extend_hub_uri():
             assert expected_output == output
 
 
+@pytest.mark.parametrize(
+    "regex_list,value,expected",
+    [
+        ([r"^.{0,9}$", r"^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"], "blabla123", True),
+        ([r"^.{0,6}$", r"^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"], "blabla123", False),
+        ([r"^.{0,6}$", r"^[a-z0-9]([-a-z0-9]*[a-z0-9])?$"], "bla^%", False),
+        ([r"^.{0,6}$", r"^a...e$", r"ab*"], "abcde", True),
+        ([r"^.{0,6}$", r"^a...e$", r"ab*"], "abababe", False),
+        ([r"^.{0,6}$", r"^a...e$", r"ab*"], "bcea", False),
+    ],
+)
+def test_get_regex_list_as_string(regex_list, value, expected):
+    match = re.match(get_regex_list_as_string(regex_list), value)
```

Comment:
Would be useful to make sure the output from the method is also as expected
```suggestion
    regex_condition = get_regex_list_as_string(regex_list)
    assert regex_condition == expected_regex_condition
    match = re.match(regex_condition, value)
    assert match if expected else match is None
```